### DISCUSSION TITLE
Sharing Collection & BeatMaps pack (Supporter only)

### DISCUSSION THREAD

**Discussion by Biokax** [ppy/osu#15205] [Ideas] [answered]
discussed in
`https://osu.ppy.sh/community/forums/topics/1443579?n=1`

- The idea is here to have a feature for Supporter only, which able to share collections.
for example on the website a new collection page that can be seen by everyone but Supporters can access to download all 
beatmaps in each collections.

- Another thing that is similar to the previus one is to create BeatMaps pack (maybe by artists or by years, need to be debated) that can only be download buy Osu! Supporters.

Enable supporters only to access download of beatmaps pack is a good thing to make people continue to buy supporter because the fact that now everyone can fast download single beatmap in game make it less atractive.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/10192#issuecomment-695217511

**Reply by Biokax** [reply]
I understand for the collection part, but what about the beatmaps pack ? I would be a great feature for supporters

**Reply by bdach** [reply]
https://github.com/ppy/osu/discussions/14328

if you're asking to be able to create your own beatmap pack then that may be a different thing from that discussion
